com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -940,7 +940,7 @@ private static ShaderFeatures GetSupportedShaderFeatures(UniversalRenderPipeline
                 if (renderer is UniversalRenderer)
                 {
                     UniversalRenderer universalRenderer = (UniversalRenderer)renderer;
-                    if (universalRenderer.renderingMode == RenderingMode.Deferred)
+                    if (universalRenderer.renderingModeRequested == RenderingMode.Deferred)
```

Comment:
Could you add info why this needs to be changed?

Reply:
This was to make it more clear to understand. RenderingMode is the rendering mode settings and then there was another RenderingModeActual which was the one that is the actual one used and considered when falling back. The renderinMode was renamed to renderingModeRequested to be more clear this is the requested mode that comes from settings but not necessary the mode that is used.

Reply:
Should I add it to the changelog, maybe?

Reply:
These are internal / private. No reason to mention in changelog.